pkgs/by-name/gf/gf/package.nix

```
@@ -1,45 +1,58 @@
-{ lib
-, stdenv
-, makeWrapper
-, fetchFromGitHub
-, libX11
-, pkg-config
-, gdb
-, freetype
-, freetypeSupport ? true
-, extensions ? [ ]
+{
+  lib,
+  stdenv,
+  makeWrapper,
+  fetchFromGitHub,
+  libX11,
+  pkg-config,
+  gdb,
+  freetype,
+  freetypeSupport ? true,
+  withExtensions ? true,
+  extraFlags ? "",
+  pluginsFile ? null,
 }:
 
 stdenv.mkDerivation {
   pname = "gf";
-  version = "unstable-2023-08-09";
+  version = "unstable-2024-08-21";
```

```suggestion
  version = "0-unstable-2024-08-21";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning

---

Done